src/main/kotlin/com/cjbooms/fabrikt/util/KaizenParserExtensions.kt

```diff
@@ -15,14 +16,14 @@ import java.net.URI
 
 object KaizenParserExtensions {
 
-    private val invalidNames =
-        listOf(
```

Comment:
**praise:** some refactoring along the way?

Reply:
Yes :)